src/serialization/AOF/rai_aof_rewrite.c

```diff
@@ -0,0 +1,105 @@
+#include "rai_aof_rewrite.h"
+
+void RAI_AOFRewriteTensor(RedisModuleIO *aof, RedisModuleString *key, void *value) {
+    RAI_Tensor *tensor = (RAI_Tensor *)value;
+
+    char *dtypestr = NULL;
+    Tensor_DataTypeStr(RAI_TensorDataType(tensor), &dtypestr);
+
+    char *data = RAI_TensorData(tensor);
+    long long size = RAI_TensorByteSize(tensor);
+
+    long long ndims = RAI_TensorNumDims(tensor);
+
+    RedisModuleString *dims[ndims];
+
+    for (long long i = 0; i < ndims; i++) {
+        dims[i] = RedisModule_CreateStringFromLongLong(RedisModule_GetContextFromIO(aof),
+                                                       RAI_TensorDim(tensor, i));
+    }
+
+    RedisModule_EmitAOF(aof, "AI.TENSORSET", "scvcb", key, dtypestr, dims, ndims, "BLOB", data,
```

Comment:
What does the "scvcb" stands for?

Reply:
see `robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int *argcp, int *flags, va_list ap)` in Redis repo

One input of the function `RedisModule_EmitAOF` is `fmt` which is the string `scvcb`, after this input you have var args input, and the  `RedisModule_EmitAOF function calls `moduleCreateArgvFromUserFormat` to know how to handle the inputs, according to the chars in `fmt`